hazelcast/src/main/resources/cluster.sh

```diff
@@ -131,6 +131,33 @@ if [ "$OPERATION" = "change" ]; then
     exit 0
 fi
 
+if [ "$OPERATION" = "force-start" ]; then
+    echo "You are force starting the cluster. Please make sure you provide valid user/pass."
```

Comment:
`You are force starting the cluster`. Any better explanation for this @mdogan @metanet ?